virt/virt-4-8-release-notes.adoc

```diff
@@ -72,7 +72,7 @@ The SVVP Certification applies to:
 
 //CNV-12269 When a VM frees space on a virtual disk the discard requests are passed to the underlying storage device which may free up real storage capacity.
 
-//CNV-12270 CDI can now automatically choose preferred accessMode and volumeMode settings when importing VM disk images.
+* You can now specify data volumes using the storage API. The storage API, unlike the PVC API, allows the system to optimize 'accessModes', 'volumeMode' and storage capacity when allocating storage.
```

Comment:
s/using/by using

I think that accessModes and volumeMode should be in backticks (like you had them before).

There should be a comma after `volumeMode`

Reply:
Done